#1469 Fix 10 Style, 1 Duplication issues in multiple files

Nockolay23 opened the pull request:
Fixed 10 styles in AddTeacher.cs file, 1 duplication problems in multiple AddTeacher.cs and AddAcademicDegree.cs files

Nockolay23 committed:
#1469

Fixed 10 styles in AddTeacher.cs file, 1 duplication problems in multiple AddTeacher.cs and AddAcademicDegree.cs files

Nockolay23 committed:
#1469

Fixed 10 Style in file AddTeacher.cs , 1 Duplication issues in multiple files AddAcademicDegree.cs and AddTeacher.cs

Nockolay23 committed:
#1469

Fixed 10 Style in file AddTeacher.cs, 1 Duplication issues in multiple files AddTeacher.cs and AddAcademicDegree.cs

Nockolay23 committed:
#1469

Fixed 10 styles in AddTeacher.cs file, 1 duplication problems in multiple AddTeacher.cs and AddAcademicDegree.cs files

YuriSilenok commented on TimetableOfClasses/AddTeacher.cs:
```diff
@@ -131,15 +130,21 @@ private bool Add()
         /// <param name="e"></param>
         private void AddTeacher_Load(object sender, EventArgs e)
         {
-
```
может стоило полностью дропнуть этот метод, раз он не используется?

YuriSilenok commented on TimetableOfClasses/AddTeacher.cs:
```diff
@@ -131,15 +130,21 @@ private bool Add()
+                R.Text = PeriodLetterToUpper(R.Text);
+            }
+        }
+        private static object SelectionOfLetters(object sender, EventArgs e)
```
почему object возвращается, а не более конкретный тип дапнных?

Nockolay23 committed:
#1469

Выкинул не нужный метод, поменял возвращаемый тип данных в методе SelectionOfLetters. Исправлено 10 стилей в файле AddTeacher.cs, 1 проблема с дублированием в нескольких файлах AddTeacher.cs и AddAcademicDegree.cs
